api/v1beta1/etcdadmconfig_types.go

```diff
@@ -97,8 +97,64 @@ type BottlerocketConfig struct {
 	// BootstrapImage specifies the container image to use for bottlerocket's bootstrapping
 	BootstrapImage string `json:"bootstrapImage"`
 
+	// AdminImage specifies the admin container image to use for bottlerocket.
+	AdminImage string `json:"adminImage"`
+
+	// ControlImage specifies the control container image to use for bottlerocket.
+	ControlImage string `json:"controlImage"`
+
 	// PauseImage specifies the image to use for the pause container
 	PauseImage string `json:"pauseImage"`
+
+	// CustomHostContainers adds additional host containers for bottlerocket.
+	// +optional
+	CustomHostContainers []BottlerocketHostContainer `json:"customHostContainers,omitempty"`
+
+	// CustomBootstrapContainers adds additional bootstrap containers for bottlerocket.
+	// +optional
+	CustomBootstrapContainers []BottlerocketBootstrapContainer `json:"customBootstrapContainers,omitempty"`
+}
+
+// BottlerocketHostContainer holds the host container setting for bottlerocket.
+type BottlerocketHostContainer struct {
+	// Name is the host container name that will be given to the container in BR's `apiserver`
+	// +kubebuilder:validation:Required
+	Name string `json:"name"`
```

Comment:
Name, Image, and UserData can technically be shared between the two structs as a separate one that is inline right? It will help us avoid having to change those fields in two places

Reply:
yea.. i don't feel strongly abt combining them just bc they share same fields - besides this is consistent w the capi BR fields. i'd prefer to leave it as it is unless we feel necessary to change it